### DISCUSSION TITLE
Assistance Needed with missing WireUI V1 Documentation

### DISCUSSION THREAD

**Discussion by OrakFR** [wireui/wireui#975] [Q&A] [answered]
Hello,

I’m currently working with [WireUI V1](https://v1.wireui.dev/docs/select) and attempting to follow the links for implementing the following scopes:

- [Async Search scope](https://github.com/wireui/old-docs/blob/main/app/Http/Controllers/Api/Users/Index.php#L19C21-L19C21)
- [Async Selected scope](https://github.com/wireui/old-docs/blob/main/app/Http/Controllers/Api/Users/Index.php#L25)

However, it seems that the [old-docs](https://github.com/wireui/old-docs) repository is no longer available or may have been moved.

Could you @PH7-Jack @joaopalopes24 or anyone else direct me to where I can find this information? 

Additionally, if possible, could someone share the code to implement the selected scope for V1?

**Note:** I am still using V1 due to customizations I've made, including publishing the WireUI config. Unfortunately, I currently don’t have the time to upgrade to V2.

Thank you for your help!

**Comment by joaopalopes24** [comment, marked as answer]
It is similar to version 2.

https://github.com/wireui/docs/blob/main/src/Examples/UserController.php
https://github.com/wireui/docs/blob/main/src/Examples/UserControllerTest.php

**Reply by OrakFR** [reply]
Thanks
